Add: RGB keyword

Tigrr18 opened the pull request:
Add NM keyword for RGB

vigliensoni commented:
@Tigrr18 before merging your contribution you must work on a few things:

- [ ] your contribution should go inside the `keywords` folder
- [ ] you must use at least three sources, from which at least two have to be from academic sources
- [ ] fix your citation style. in Chicago author-date, the book's name is italicized and the year goes after the author. For example, the third one should be

Ibraheem, Noor, Mokhtar Mohammed Hasan, Rafiqul Zaman Khan, and Pramod Kumar Mishra. 2012. “Understanding Color Models: A Review.” _ARPN Journal of Science and Technology_ 2 (3): 265–275.

- [ ] improve the content. create a link of your topic to new media. why is it relevant to new media? where is it used?

Tigrr18 commented:
I have added two citations by academic sources (one book, one article). I have also fixed (hopefully) the citation format. I have added the image required for tonight. (the one page collage). I have also added information as well as links to new media art.
